src/MentionsTextInput.js

```diff
@@ -371,6 +370,31 @@ export default class MentionsTextInput extends Component {
     }
   }
 
+  setSelection(start, end = start) {
```

Comment:
what are `start` and `end`?

Reply:
the start and end of the selection!

Reply:
eg: 1234 -> you could select the chars from 0, 3

Reply:
oh so these are indices, that wasn't too clear, but was able to gather from context.  Might be useful to use flowtypes to say that they're numbers, e.g. `start: number, end: number=start`

Reply:
Or maybe call it `setSelectionFromIndices`?

Reply:
its called selection its a prop

Reply:
the name is consistent with the name of the prop and the name of the passed param to `onSelectionChange`

Reply:
ie `selection: {start, end}` is what RN has set as the naming convention

Reply:
gotcha